src/main/java/com/mindee/parsing/common/field/LineItemField.java

```diff
@@ -0,0 +1,12 @@
+package com.mindee.parsing.common.field;
+
+import java.util.HashMap;
+
+/**
+ * Represent a single line in a field printable as an rST table.
+ */
+public abstract class LineItemField extends BaseField {
```

Comment:
Since lineitemfield is only used to abstract behaviour/contract and not inherit them, does it make sense to have it as an interface?

Reply:
It does, but `BaseField` itself is not an interface...

It probably should be reworked as interfaces, for other reasons as well, but I feel this work should belong in a separate PR.